### ISSUE TITLE
Unable to insert data and files for the illustration category (person, publication, and attachment)

### ISSUE THREAD

**Issue by GiuliaBruna** [martijnbentum/region#29]
When inputting data pertaining to the Illustration category, I cannot successfully complete the task. I cannot attach files and link the illustration with its Person and Publication. When clicking Save, it doesn't save anything. Thank you for your help.

**Comment by GiuliaBruna** [comment]
I tried both your suggestions, but when I click on save it does not save and it does not seem to have uploaded anything and doesn't save the entry either. I tried opening the database both from Chrome and Safari. I always managed to upload illustrations before these couple of days. Thank you.

**Comment by GiuliaBruna** [comment]
> I tried both your suggestions, but when I click on save it does not save and it does not seem to have uploaded anything and doesn't save the entry either. I tried opening the database both from Chrome and Safari. I always managed to upload illustrations before these couple of days. Thank you.

I tried again and I think I found what the problem/bug was and ultimately managed to attach the illustrations. The issue seems to be with Frontispiece illustrations: if I write Frontispiece in the page number, it actually allows me to write it, but then doesn't save. I circumvented the issue by writing page 1 or leaving it blank for Frontispiece Illustrations. So if I put a number in the page number I can actually save and upload the illustrations no problem. Not sure what's best to do for those types of illustrations, perhaps we can have a quick chat about it at the next meeting. Thank you.
